tests/registry/stacks/go/2.1.0/devfile.yaml

```diff
@@ -11,4 +11,4 @@ metadata:
     - Go
   version: 2.1.0
 parent:
-  uri: https://registry.devfile.io/devfiles/go/2.1.0
+  uri: https://raw.githubusercontent.com/devfile/registry/main/stacks/go/2.1.0/devfile.yaml
```

Comment:
is this change mean to be pushed?

Reply:
Yes, the test was failing due to the previous link. It seems that due to recent changes the URI of this format is not supported anymore. The reason is we are now using [DownloadGitRepoResources](https://github.com/devfile/library/blame/main/pkg/devfile/parser/utils.go#L44) which expects a git provider link instead of the registry one.